h/static/scripts/test/streamer-test.js

```diff
@@ -74,41 +75,39 @@ describe('streamer', function () {
       websocketUrl: 'ws://example.com/ws',
     };
 
-    streamer = proxyquire('../streamer', {
+    Streamer = proxyquire('../streamer', {
       './websocket': FakeSocket,
     });
   });
 
   it('should not create a websocket connection if websocketUrl is not provided', function () {
     fakeSettings = {};
     createDefaultStreamer();
+    activeStreamer.connect();
```

Comment:
In this PR, the streamer still connects when it is constructed, so I don't think the call to `connect()` is needed here? Although it is harmless.